crates/db_views/src/tagline_view.rs

```diff
@@ -0,0 +1,38 @@
+use crate::structs::TaglineView;
+use diesel::{result::Error, ExpressionMethods, QueryDsl};
+use diesel_async::RunQueryDsl;
+use lemmy_db_schema::{
+  newtypes::LocalSiteId,
+  schema::tagline,
+  source::tagline::Tagline,
+  utils::{get_conn, limit_and_offset, DbPool},
+};
+
+impl TaglineView {
+  pub async fn list(
+    pool: &mut DbPool<'_>,
+    for_local_site_id: LocalSiteId,
+    page: Option<i64>,
+    limit: Option<i64>,
+  ) -> Result<Vec<Self>, Error> {
+    let conn = &mut get_conn(pool).await?;
+    let (limit, offset) = limit_and_offset(page, limit)?;
+    let taglines = tagline::table
+      .filter(tagline::local_site_id.eq(for_local_site_id))
+      .order(tagline::id)
```

Comment:
Order by the published date descending, so it shows newest first.

Reply:
`TaglineView` was removed. New `list` function can be found [here](https://github.com/LemmyNet/lemmy/pull/4580/files#diff-4118e2f1f35bfd0d48a41dabd4366577863e150820ea6ab5372e73a004271e95R68)